core/src/main/java/org/geysermc/geyser/text/MinecraftLocale.java

```diff
@@ -169,23 +198,21 @@ private static boolean loadLocale(String locale) {
                 langMap.put(entry.getKey(), entry.getValue().asText());
             }
 
-            String bedrockLocale = locale.toLowerCase(Locale.ROOT);
-            if (bedrockLocale.equals("no_no")) {
-                // Store this locale under the Bedrock locale so we don't need to do this check over and over
-                bedrockLocale = "nb_no";
-            }
-
-            // Insert the locale into the mappings
-            LOCALE_MAPPINGS.put(bedrockLocale, langMap);
+            return langMap;
+        } catch (IOException e) {
+            throw new AssertionError(GeyserLocale.getLocaleStringLog("geyser.locale.fail.file", locale, e.getMessage()));
```

Comment:
Wouldn't the 
```suggestion
            throw new AssertionError(GeyserLocale.getLocaleStringLog("geyser.locale.fail.json", locale, e.getMessage()));
```
fit better? Atleast judging by the previous implementation. Then again, due to how both are combined, it may make sense to catch different exceptions

Reply:
Added a separate catch